portiaui/app/components/jobq-badge.js

```diff
@@ -0,0 +1,18 @@
+import Ember from 'ember';
+const { computed } = Ember;
+
+export default Ember.Component.extend({
+    jobQ: Ember.inject.service(),
+
+    jobs: computed.alias('jobQ.jobs'),
+
+    jobCount: computed('jobs.count', function() {
```

Comment:
will recomputing only when count changes be enough to catch all changes? what if one spider ends and another starts between polls